### ISSUE TITLE
Using pronto gem preventing Docker local development environment set up

### ISSUE THREAD

**Issue by yaningo** [circleci/circleci-docs#5270]
# Description
Following instructions to [set up a local Development with Docker](https://github.com/circleci/circleci-docs/blob/master/docs/local-development.md#1-local-development-with-docker-recommended) fails with:

```
jekyll_1  | Installing rugged 1.0.1 with native extensions
jekyll_1  | Gem::Ext::BuildError: ERROR: Failed to build gem native extension.
jekyll_1  | 
jekyll_1  |     current directory: /usr/local/bundle/gems/rugged-1.0.1/ext/rugged
jekyll_1  | /usr/local/bin/ruby -I /usr/local/lib/ruby/2.7.0 -r
jekyll_1  | ./siteconf20210426-7-mmxbs5.rb extconf.rb
jekyll_1  | checking for gmake... no
jekyll_1  | checking for make... yes
jekyll_1  | checking for cmake... no
jekyll_1  | ERROR: CMake is required to build Rugged.
jekyll_1  | *** extconf.rb failed ***
jekyll_1  | Could not create Makefile due to some reason, probably lack of necessary
jekyll_1  | libraries and/or headers.  Check the mkmf.log file for more details.  You may
jekyll_1  | need configuration options.
```
```
jekyll_1  | An error occurred while installing rugged (1.0.1), and Bundler cannot continue.
jekyll_1  | Make sure that `gem install rugged -v '1.0.1' --source 'https://rubygems.org/'`
jekyll_1  | succeeds before bundling.
jekyll_1  | 
jekyll_1  | In Gemfile:
jekyll_1  |   pronto was resolved to 0.11.0, which depends on
jekyll_1  |     rugged
```
# Location
This seems to have been introduced by [this pull-request](https://github.com/circleci/circleci-docs/pull/5226)

Removing this from the [Gemfile](https://github.com/circleci/circleci-docs/blob/ae6216086b634d89ded4a2a690ab9a649ac65cd9/Gemfile) fixes the issue:
```
group :test, :development do
  gem 'pronto'
  gem 'pronto-markdownlint'
end
```

# Steps to Reproduce
Follow instructions to [set up a local Development with Docker](https://github.com/circleci/circleci-docs/blob/master/docs/local-development.md#1-local-development-with-docker-recommended)

**Comment by teesloane** [comment]
@teka23 might have some ideas on how to fix this.

For now I would recommend running jekyll locally:

```sh
# in circleci-docs/jekyll
bundle exec jekyll serve -Iw
```

If you run into issues, you may need to run a bundle clean/update/etc:

`bundle update jekyll; bundle exec jekyll clean; npm run webpack-prod; bundle exec jekyll serve -Iw`

**Comment by teesloane** [comment]
This has been fixed by #5382 .
